target/linux/octeon/image/Makefile

```diff
@@ -29,6 +29,15 @@ define Device/generic
 endef
 TARGET_DEVICES += generic
 
+ITUSROUTER_CMDLINE:=console=ttyS0,115200, root=/dev/mmcblk1p2 rootfstype=squashfs,ext4,f2fs rootwait
+define Device/itus_shield-router
+  DEVICE_VENDOR := Itus Networks
+  DEVICE_MODEL := Shield Router
```

Comment:
The thing is, this device is called Itus Shield (maybe Pro) and in any role that's still the same device, the only difference is moved role switch. I would suggest drop the router suffix from whole commit and inform users in the message, to put the switch position to "R".

Reply:
Valid point. Same should be done for the other strings then, i.e. itus_shield and itus,shield.

Reply:
If I do that, I lose the ability to locally support the gateway and bridge slots..  It was determined that official openwrt support could be limited to the "router" slot, but continuing to have the format like it is would allow me to support the other two.

Reply:
Also, I think the Itus Shield Pro (which was supposed to be developed) never made it to the public.  It was supposed to include WiFi support (which the Shield doesn't).  the DeviWiki on it is confusing because they link to the "Shield Pro" in the middle, but the right column has the actual tech stats for the device :/

Reply:
Well the roles are simply created by applying configuration. If You know the position of the switch, You can add the detection to `target/linux/octeon/base-files/lib/preinit/01_sysinfo` and apply the configuration to the interfaces by the name specified there. If it also means, adding particular packages/applications then leave it as is.

Reply:
@tmn505 We had something like that before, but it became a mess as it would have required us to create three separate "devices" after all. So, we kind of chose the current setup as kind of a compromise.

Reply:
Ok, then keep it as is.